src/pybind/mgr/dashboard/services/cephfs.py

```diff
@@ -298,3 +298,12 @@ def statfs(self, path) -> dict:
             rfiles = int(self.cfs.getxattr(path, 'ceph.dir.rfiles'))
             rsubdirs = int(self.cfs.getxattr(path, 'ceph.dir.rsubdirs'))
         return {'bytes': rbytes, 'files': rfiles, 'subdirs': rsubdirs}
+
+    def rename(self, src_path, dst_path) -> None:
```

Comment:
this should be renamed too, folks might confuse this with the existing rename API to rename CephFS volume.

Reply:
done